src/caw/client/cpp_client/caw_client.cpp

```diff
@@ -196,6 +196,44 @@ bool CawClient::GetProfile(const std::string &username) {
   return profileSuccessBool;
 }
 
+bool CawClient::StreamHashtag(const std::string &hashtag) {
+  std::cout << "Streaming Caws with hashtag: #" << hashtag << std::endl;
+  HashtagRequest hashtag_request;
+  hashtag_request.set_hashtag(hashtag);
+  Any request;
+  request.PackFrom(hashtag_request);
+  // Reads and prints the Caw
+  std::function<void(EventReply)> callback = [](EventReply event_reply) {
+    HashtagReply reply;
+    Any(event_reply.payload()).UnpackTo(&reply);
```

Comment:
Just curious, any reason why you're casting this to type Any?

Reply:
An Event Reply contains an Any (named payload), which contains a Hashtag Reply, so this is a shortcut to go from EventReply directly to HashtagReply (since I do not care about the Any object.  I will add a comment next to this explaining it since I do see how it can be confusing.